charts/ceph-csi-cephfs/templates/provisioner-statefulset.yaml

```diff
@@ -61,30 +63,23 @@ spec:
 {{ toYaml .Values.provisioner.attacher.resources | indent 12 }}
 {{- end }}
         - name: csi-cephfsplugin
-          securityContext:
-            privileged: true
-            capabilities:
-              add: ["SYS_ADMIN"]
-            allowPrivilegeEscalation: true
           image: "{{ .Values.nodeplugin.plugin.image.repository }}:{{ .Values.nodeplugin.plugin.image.tag }}"
           imagePullPolicy: {{ .Values.nodeplugin.plugin.image.pullPolicy }}
-          args :
+          args:
             - "--nodeid=$(NODE_ID)"
             - "--type=cephfs"
             - "--controllerserver=true"
+            - "--pidlimit=-1"
+{{- if .Values.provisioner.grpcMetrics.enabled }}
+            - "--metricsport={{ .Values.nodeplugin.grpcMetrics.containerPort }}"
+            - "--metricspath=/metrics"
+            - "--enablegrpcmetrics=true"
+{{- end }}
             - "--endpoint=$(CSI_ENDPOINT)"
             - "--v=5"
             - "--drivername=$(DRIVER_NAME)"
-            - "--metadatastorage=k8s_configmap"
-            - "--pidlimit=-1"
-            - "--metricsport=8091"
-            - "--metricspath=/metrics"
-            - "--enablegrpcmetrics=false"
+            - "--containerized=true"
           env:
-            - name: POD_IP
```

Comment:
is this removed?

Reply:
Nope, github is weirding out on the diff here:
https://github.com/wilmardo/ceph-csi/blob/improve-helm/charts/ceph-csi-cephfs/templates/provisioner-statefulset.yaml#L121